html/index.html.haml

```diff
@@ -1,388 +1,21 @@
-            :meta => "description.meta.changed_managed_files", |
-            :singular => "'file'", |
-            :plural => "'files'", |
-            :count => "description.changed_managed_files.files.length" |
-          } |
-          .row.scope_content.collapse.in
-            .col-xs-1
-            .col-xs-11
-              %table.table.table-striped.table-hover.table-condensed.files-table
-                %thead
-                  %tr
-                    %th Name
-                    %th Package Name
-                    %th Package Version
-                    %th Changes
-                    %th Mode
-                    %th User
-                    %th Group
-                %tbody
-                  %tr{"ng-repeat" => "file in description.changed_managed_files.files | filter:query"}
-                    %td{"ng-switch" => true, "on" => "file.downloadable"}
-                      %a.file-download{"ng-switch-when" => "true", "href" => "#"}
-                        {{file.name}}
-                      %span{"ng-switch-default" => true}
-                        {{file.name}}
-                    %td {{file.package_name}}
-                    %td {{file.package_version}}
-                    %td
-                      {{file.changes.join(",")}}
-                      %span{"ng-show" => "file.error_message", :title => "{{file.error_message}}"} (details)
-                    %td {{file.mode}}
-                    %td {{file.user}}
-                    %td {{file.group}}
-
-    %script#scope_unmanaged_files.partial{:type => "text/ng-template"}
-      %div{"ng-show" => "description.unmanaged_files.files"}
-        %a.scope_anchor{:id => "unmanaged_files"}
-        #unmanaged_files_container.scope{"data-scope" => "unmanaged_files"}
-          %scope-header{ |
-            :summary => "'#{scope_help('unmanaged_files')}'", |
-            :logo => "'assets/logo-unmanaged-files.png'", |
-            :title => "'Unmanaged Files'", |
-            :meta => "description.meta.unmanaged_files", |
-            :singular => "'file'", |
-            :plural => "'files'", |
-            :count => "description.unmanaged_files.files.length" |
-          } |
-          .row.scope_content.collapse.in
-            .col-xs-1
-            .col-xs-11
-              %table.table.table-striped.table-hover.table-condensed.files-table
-                %thead
-                  %tr
-                    %th Name
-                    %th Type
-                %tbody
-                  %tr{"ng-repeat" => "file in description.unmanaged_files.files | filter:query"}
-                    %td{"ng-switch" => true, "on" => "file.downloadable"}
-                      %a.file-download{"ng-switch-when" => "true", "href" => "#"}
-                        {{file.name}}
-                      %span{"ng-switch-default" => true}
-                        {{file.name}}
-                    %td {{file.type}}
-
+    %script{:src => "assets/jquery.searcher.min.js"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace